client/webserver/site/src/js/forms.js

```diff
@@ -1003,11 +1003,3 @@ export function bind (form, submitBttn, handler) {
 function isTruthyString (s) {
   return s === '1' || s.toLowerCase() === 'true'
 }
-
-function unitInfo (xc, assetID) {
-  const dexAsset = xc.assets[assetID]
-  if (dexAsset && dexAsset.unitInfo.conventional.conversionFactor > 0) return dexAsset.unitInfo
```

Comment:
What was this >0 for?  That's a funny check.

Reply:
The `UnitInfo` is a value instead of a pointer in many places, notable `dex.Asset`. Similarly, the `Conventional Denomination` field is a value. So I was thinking it wouldn't be crazy for an incorrect configuration to have a `conversionFactor` of zero. I'm not checking for the condition in the combined `Application.unitInfo` method though.